test/Test/Toml/Parsing/Unit.hs

```diff
@@ -113,22 +113,24 @@ spec_Parser = do
             boolFailOn "fAlSE"
 
     describe "doubleP" $ do
-        it
-                "can parse a number which consists of an integral part, and a fractional part"
+        it "can parse a number which consists of an integral part, and a fractional part"
             $ do
                   parseDouble "+1.0"   1.0
                   parseDouble "3.1415" 3.1415
                   parseDouble "0.0"    0.0
                   parseDouble "-0.01"  (-0.01)
-        it
-                "can parse a number which consists of an integral part, and an exponent part"
+        it "can parse a number which consists of an integral part, and an exponent part"
             $ do
                   parseDouble "5e+22" 5e+22
                   parseDouble "1e6"   1e6
                   parseDouble "-2E-2" (-2E-2)
-        it
-                "can parse a number which consists of an integral, a fractional, and an exponent part"
+        it "can parse a number which consists of an integral, a fractional, and an exponent part"
             $ parseDouble "6.626e-34" 6.626e-34
+        it "can parse a number with underscores"
+            $ do
```

Comment:
I think it's better to write `$ do` on the same line as `it " "` like:

```haskell
it "..." $ do
```
No need to have extra line